spring-cloud-data-admin/src/main/java/org/springframework/cloud/data/admin/config/AdminConfiguration.java

```diff
@@ -86,7 +86,12 @@ public ModuleRegistry moduleRegistry() {
 	protected static class LocalConfig {
 
 		@Bean
-		public ModuleDeployer moduleDeployer(ModuleLauncher moduleLauncher) {
+		public ModuleDeployer longRunningModuleDeployer(ModuleLauncher moduleLauncher) {
```

Comment:
Those names map to the 2 consumers of the beans: StreamController and TaskController

IMO, the Qualifier value should be mapped to the target of the bean injection more than the source. We get more opinionated and domain-specific as we move up the layers. Do you have anything specific in mind for how the spring-cloud-data domain itself would be extended beyond "stream" and "task". I don't want to prematurely over-generalize.